doc/nrf/gs_ins_mac.rst

```diff
@@ -64,3 +64,39 @@ Install Homebrew and install the required tools using the ``brew`` command line
 .. include:: gs_ins_windows.rst
    :start-after: buildenv_start
    :end-before: buildenv_end
+
+.. _additional_mac_setup:
+
+Setting up executables on macOS
+===============================
+
+If you start SES on macOS by running the file :file:`bin/emStudio`, make sure to complete the following steps:
+
+1. Specify the path to all executables under :guilabel:`Tools` -> :guilabel:`Options` (in the :guilabel:`nRF Connect` tab).
+   The global PATH on macOS is used only when :ref:`using the command line environment <build_environment_cli_mac>`.
```

Comment:
No, this is wrong. It is used when starting SES from a command prompt.

Reply:
Rephrased.